kong/pdk/private/node.lua

```diff
@@ -0,0 +1,86 @@
+local utils = require "kong.tools.utils"
+local pl_file = require "pl.file"
+local pl_path = require "pl.path"
+local pl_dir = require "pl.dir"
+
+local ngx = ngx
+
+local cached_node_id
+
+local function node_id_filename(prefix)
+  return pl_path.join(prefix, "kong.id")
+end
+
+
+local function initialize_node_id(prefix)
+  if not pl_path.exists(prefix) then
```

Comment:
Is it an option that stores the node ID in the LMDB?

Reply:
+1. But I'm convinced that this will be a transitional solution and we don't need to spend too much effort on it.

Reply:
Actually, LMDB is not available in the init phase.